portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ActiveIndividuals.testcase

```diff
@@ -166,9 +166,45 @@ definition {
 	@ignore = "true"
 	@priority = "5"
 	test GraphShowsActivitiesFromTodayWhenSwitchedTo24HrFilter {
+		ACUtils.createDMAndAddToPage();
+
+		var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");
+
+		JSONUser.addUser(
+			userEmailAddress = "ac@liferay.com",
+			userFirstName = "ac",
+			userLastName = "ac",
+			userScreenName = "ac");
+
+		ACDXPSettings.syncNewContactData();
+
+		User.logoutAndLoginPG(
+			userLoginEmailAddress = "ac@liferay.com",
+			userLoginFullName = "ac ac");
+
+		for (var n : list "1,2,3,4") {
+			Navigator.gotoSitePage(
+				pageName = "AC Page",
+				siteName = "Site Name");
 
-		// TODO LRAC-8139 GraphShowsActivitiesFromTodayWhenSwitchedTo24HrFilter pending implementation
+			DMNavigator.gotoDocumentCP(dmDocumentTitle = "DM AC Title");
+		}
+
+		ACUtils.closeAllSessions();
+
+		Pause(locator1 = "30000");
+
+		ACUtils.launchAC();
+
+		ACProperties.switchProperty(propertyName = "Liferay DXP Combined Property");
```

Comment:
hey @ViniLopes20 , I think you forgot to pass the propertyname value instead of "Liferay DXP..."